### ISSUE TITLE
How likely can this run on other Acer Chrombooks?

### ISSUE THREAD

**Issue by ChaseFlorell** [codestarterorg/ubuntu-chromebook-installer#24]
Is this super specific to one model or can it run on others? I'm looking at doing around a $400 CB but would love to be able to do this approach.

**Comment by mojombo** [comment]
This installer is specific to the Acer C720, but much of it can probably be reused on other Chromebooks with a similar architecture. There are other, more generic installers available elsewhere, too. Did you know what model you're thinking about?

**Comment by ChaseFlorell** [comment]
Not entirely sure the model, but something available at my [local shop](http://www.memoryexpress.com/Search/Products?Search=chromebook). Also open to recommendations, but the C720 is hard to find (as you've already discovered).

**Comment by PhiTauBill** [comment]
Will this work on an Acer 670 like this one (http://www.monoprice.com/product?p_id=14213) which is widely available as a refurb at a reasonable price?

**Comment by mojombo** [comment]
@PhiTauBill No, I don't think so. If I recall, there is a problem with the bootloader on that model that prevents it from booting into the secondary partition. Sorry!
